terraform/environments/core-network-services/locals.tf

```diff
@@ -30,6 +30,50 @@ locals {
     "core-network-services-live_data-attachment",
   ]
 
+  active_tgw_all_attachments = [
```

Comment:
Could we get this list dynamically so we don't need to manually update it?  It might need to be a concat of a few different things